modules/rosa-sts-aws-requirements-creating-association.adoc

```diff
@@ -7,14 +7,14 @@
 [id="rosa-associating-account_{context}"]
 = Linking your AWS account
 
-You can link your AWS account to existing IAM roles by using the `rosa` CLI.
+You can link your AWS account to existing IAM roles by using the {product-title} (ROSA) CLI (`rosa`).
 
 .Prerequisites
 
 * You have an AWS account.
 * You are using {cluster-manager-url} to create clusters.
 * You have the permissions required to install AWS account-wide roles. See the "Additional resources" of this section for more information.
-* You have installed and configured the latest AWS (`aws`) and ROSA (`rosa`) CLIs on your installation host.
+* You have installed and configured the latest AWS (`aws`) and the ROSA (`rosa`) CLIs on your installation host.
```

Comment:
This change is unnecessary, and makes the sentence less correct, as it now implies that there is more than one ROSA CLI.

You could change it to "the latest AWS CLI (`aws`) and the latest ROSA CLI (`rosa`)"

Reply:
Revoking the change.